# Broken App Issues
. In the catch block- next(err) without defining the err parameter
    update it to catch (err) and use it within the block. 
    not enough error handling. 
    